concordia/templates/transcriptions/campaign_list.html

```diff
@@ -13,28 +13,24 @@
 {% endblock breadcrumbs %}
 
 {% block main_content %}
-<div class="container-fluid py-3">
-    <div class="row">
-        <div class="d-flex flex-wrap col-12 mb-3 justify-content-around">
-            {% for campaign in campaigns %}
-                <div class="bg-lightest-gray w-100 mb-3">
-                    <div class="row">
-                        <div class="col-12 col-md-8 px-3">
-                            <a href="{% url 'transcriptions:campaign-detail' campaign.slug %}">
-                                <h1 class="campaign-title">{{ campaign.title }}</h1>
-                            </a>
-                            {{ campaign.short_description|linebreaksbr }}
-                        </div>
-                        <div class="col-12 col-md-4">
-                            <a href="{% url 'transcriptions:campaign-detail' campaign.slug %}">
-                            <img src="{{ MEDIA_URL }}{{ campaign.thumbnail_image }}" class="img-fluid" alt="{{ campaign.title }} image">
-                            </a>
-                        </div>
-                    </div>
+<div class="container py-3">
+    <h1 class="mb-5 sr-only">Campaigns</h1>
```

Comment:
Does this really need a margin tag if it's only for screen readers?

Reply:
I'm wondering whether we should hoist the `sr-only` H1 into the base template — just put something like this, just above the `main_content` block:

```django
        {% block page-title %}
            {% if title %}
                <h1 id="page-title" class="sr-only">{{ title }}</h1>
            {% endif %}
        {% endblock page-title%}
```

Reply:
I think most pages have correct H1 title except for this page. I wanted to show Jaime what it looks like with the H1 displayed at the top. I'll work with Jaime to see if he wants it hidden or not, and will update this.

Reply:
It’s been inconsistent, especially for things like the login or static pages. We have touched a couple but for accessibility I’m wondering whether we just make a separate task to review & consolidate it site-wide.

Reply:
Removed margin class as Jamie wants H1 to be hidden for now.